### PR TITLE
[CHI-199] Review All tests pages regarding colors

### PR DESCRIPTION
**Pull request by davidojedacl**
@mattnickles @jllr

### CODE REVIEW

File: test/chi/components/badges.pug [hunk 1]
```diff
@@ -60,7 +60,7 @@ each type in ['solid', 'outline']
     span.a-h2= `Badge / Pill / ${type}`
```
Comment (jllr) [inline test/chi/components/badges.pug]: @davidojedacl badge pills are duplicated, please remove one of them

**Comment by jllr** [comment]
@davidojedacl Horizontal Button groups / outline / default / fluid is not taking pill examples

screen shot 2019-02-27 at 12 41 21 pm

**Comment by jllr** [comment]
@davidojedacl there are conflicts in this PR, please resolve them

**Comment by davidojedacl** [comment]
done

**Pull request merged**